src/Explorer/Explorer.tsx

```diff
@@ -361,9 +356,6 @@
       };
       const connectionInfo = await this.phoenixClient.containerConnectionInfo(provisionData);
       if (connectionInfo.data && connectionInfo.data.notebookServerUrl) {
-        connectionStatus.status = ConnectionStatusType.Connected;
```

Comment:
if u remove this - where does it update to connected/

Reply:
Maintaining all status updates in phoenixclient.cs

Reply:
we have consolidated all the status updates in phoenixclient.ts